internal/autothrottle/replication/topics_legacy.go

```diff
@@ -19,8 +19,9 @@ func (tm *ThrottleManager) legacyGetTopicsWithThrottledBrokers() (TopicThrottled
 		return nil, err
 	}
 
-	// Lookup brokers with overrides set that are not a reassignment participant.
-	throttledBrokers := tm.brokerOverrides.Filter(NotReassignmentParticipant)
+	//Lookup brokers with overrides set (previously we only filtered for brokers
+	// that are not a reassignment participant).
```

Comment:
similar nit: no need to mention previous behaviour in the comment

Reply:
Fixed